pages/1.13/security/oss/iam-api/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: layout.pug
+navigationTitle:  Identity and Access Management API
+title: Identity and Access Management API
+menuWeight: 40
+excerpt: Using the DC/OS Identity and Access Management API
+
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+The Identity and Access Management API allows you to manage users through a RESTful interface.
+
+# Request and response format
+
+The API supports JSON only. You must include `application/json` as your `Content-Type` in the HTTP header, as shown below.
+
+    Content-Type: application/json
+
+# Host name and base path
+
+The host name to use will vary depending on where your program is running.
+
+* If your program runs outside of the DC/OS cluster, you should use the cluster URL. This can be obtained by launching the DC/OS web interface and copying the domain name from the browser. Alternatively, you can log in to the DC/OS CLI and type `dcos config show core.dcos_url` to get the cluster URL. In a production environment, this should be the path to the load balancer which sits in front of your masters.
```

Comment:
Rewrite `This can be obtained by launching the DC/OS web interface and copying the domain name from the browser`

with 
`This is achieved by launching the DC/OS web interface and copying the domain name from the browser.`